test/TestRoleHandlers.py

```diff
@@ -1,9 +1,8 @@
 import unittest
 
 from ansiblelint.rules import RulesCollection
-from ansiblelint.rules.UseHandlerRatherThanWhenChangedRule import (
-    UseHandlerRatherThanWhenChangedRule,
-)
+from ansiblelint.rules.UseHandlerRatherThanWhenChangedRule import \
```

Comment:
Braces must be always used instead of escaping EOL.

Reply:
Guess what, that happened when I removed black profile, the only thing changed. So now we have a good reason to use it.

Reply:
No, this only means that we still have a good reason to use the option that sets it up correctly, as I mentioned earlier: https://github.com/ansible/ansible-lint/pull/887#discussion_r455257875.